### DISCUSSION TITLE
Help Plz

### DISCUSSION THREAD

**Discussion by Crepo920** [Godiesc/firefox-gx#198] [General]
![image](https://github.com/user-attachments/assets/f5240712-6473-4c21-ae3a-32879387f298)
![image](https://github.com/user-attachments/assets/10c36bc0-cc0b-4e78-9068-7caec6cde149)
i have enter Lightgreen and it show light green as well but when i open a new tab its becoming red can i fix it or its just a bug?? should i wait for the update?

**Comment by Godiesc** [comment]
hi, for each wallpaper you have an accent color, you can see just below in your screenshot the `--general-color-dark` variable for the wallpapers.
